applicationset/utils/utils.go

```diff
@@ -25,6 +25,8 @@ import (
 	argoappsv1 "github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
 )
 
+//go:generate go run github.com/vektra/mockery/v2@v2.40.2 --name=Renderer
+
```

Comment:
I think --name=renderer or make codegen again
Currently, the go generate command generates Renderer.go, not renderer.go.